### ISSUE TITLE
Javadoc warnings

### ISSUE THREAD

**Issue by deepu105** [jhipster/generator-jhipster#3901]
We are recently seeing a lot of Javadoc warnings in the travis build. we need to fix these.

```
19 warnings
[WARNING] Javadoc Warnings
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/config/DefaultProfileUtil.java:31: warning: no @return
[WARNING] public static String getDefaultActiveProfiles(){
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/config/DefaultProfileUtil.java:47: warning: no @param for app
[WARNING] public static void addDefaultProfile(SpringApplication app) {
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/service/LabelService.java:92: warning: no @param for pageable
[WARNING] public Page<Label> search(String query, Pageable pageable) {
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestInfiniteScrollEntityResource.java:160: warning: no @param for pageable
[WARNING] public ResponseEntity<List<FieldTestInfiniteScrollEntity>> searchFieldTestInfiniteScrollEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestInfiniteScrollEntityResource.java:160: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<FieldTestInfiniteScrollEntity>> searchFieldTestInfiniteScrollEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestPagerEntityResource.java:160: warning: no @param for pageable
[WARNING] public ResponseEntity<List<FieldTestPagerEntity>> searchFieldTestPagerEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestPagerEntityResource.java:160: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<FieldTestPagerEntity>> searchFieldTestPagerEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestPaginationEntityResource.java:160: warning: no @param for pageable
[WARNING] public ResponseEntity<List<FieldTestPaginationEntity>> searchFieldTestPaginationEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/FieldTestPaginationEntityResource.java:160: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<FieldTestPaginationEntity>> searchFieldTestPaginationEntities(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/LabelResource.java:153: warning: no @param for pageable
[WARNING] public ResponseEntity<List<Label>> searchLabels(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/LabelResource.java:153: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<Label>> searchLabels(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/OperationResource.java:160: warning: no @param for pageable
[WARNING] public ResponseEntity<List<Operation>> searchOperations(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/OperationResource.java:160: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<Operation>> searchOperations(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestInfiniteScrollResource.java:167: warning: no @param for pageable
[WARNING] public ResponseEntity<List<TestInfiniteScroll>> searchTestInfiniteScrolls(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestInfiniteScrollResource.java:167: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<TestInfiniteScroll>> searchTestInfiniteScrolls(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestPagerResource.java:167: warning: no @param for pageable
[WARNING] public ResponseEntity<List<TestPager>> searchTestPagers(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestPagerResource.java:167: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<TestPager>> searchTestPagers(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestPaginationResource.java:167: warning: no @param for pageable
[WARNING] public ResponseEntity<List<TestPagination>> searchTestPaginations(@RequestParam String query, Pageable pageable)
[WARNING] ^
[WARNING] /home/travis/app/src/main/java/com/mycompany/myapp/web/rest/TestPaginationResource.java:167: warning: no @throws for java.net.URISyntaxException
[WARNING] public ResponseEntity<List<TestPagination>> searchTestPaginations(@RequestParam String query, Pageable pageable)
[WARNING] ^
```

**Comment by cbornet** [comment]
Once this is fixed, we should probably make javadoc warnings fail the build

**Comment by deepu105** [comment]
Closed  via #3904
